DSharpPlus.Test/Role-IconTest.cs

```diff
@@ -39,6 +39,7 @@ public async Task Fetch(CommandContext ctx, DiscordRole role)
             await ctx.RespondAsync(
                 $"Role: {role.Mention}\n" +
                 $"Role has icon: {role.IconHash != null}\n" +
+                $"Associated emoji: {role.Emoji}\n +" +
```

Comment:
Why "Associated"
You can either set unicode_emoji or icon

Reply:
>Questioning testbot code

In all seriousness, though, I did it because the emoji may be present regardless of if an icon is set. It's mainly just for debugging purposes, so

Reply:
ah